<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainApp</string>
  </property>
  <widget class="QWidget" name="app">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <widget class="QLineEdit" name="inputBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>441</width>
      <height>831</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="cosButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sinButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tanButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>mod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acosButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="asinButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="atanButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>atan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sqrtButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lnButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="umButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minusButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plusButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="oneButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="twoButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="threeButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>430</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fourButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fiveButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>530</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sixButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>530</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sevenButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>630</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eightButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>630</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nineButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>630</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>730</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zeroButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>730</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dotButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>730</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="equalButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>630</y>
       <width>81</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openBracketButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBracketButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="plusButton_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>530</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="answerBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="creditButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Calc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit Calc</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
